Testing/functional/tests/cypress/integration/ui/services/view.js

```diff
@@ -1,4 +1,6 @@
-const servicesTestsConstants = require("./servicesTestConstants");
+const servicesTestsConstants = {
+    servicesUrl: "/services",
+};
 const { AuthMethod } = require("../../../support/constants");
 
 describe("Authenticated Services View", () => {
```

Comment:
First, configureSettings function in cypress/support.commands.js needs to be updated to support services-services-name/enabled.

Secondly, this test does not go the services page.  It goes to the home page and then the test checks if the URL contains servicesTestsConstants.servicesUrl in Testing/functional/tests/cypress/support/constant.js but there is no serviceUrl there.

<img width="1481" alt="Screenshot 2023-03-10 at 7 24 32 AM" src="https://user-images.githubusercontent.com/58790456/224356545-7e198f10-f749-4a64-87b2-24401ff57b29.png">

<img width="1375" alt="Screenshot 2023-03-10 at 7 26 54 AM" src="https://user-images.githubusercontent.com/58790456/224356587-4ab4cdff-4d5a-4454-b4b0-9bff29fb9300.png">

Thirdly you will also need tests where organDonarRegistrationCard is enabled and disabled.

Reply:
First comment will be addressed by AB#15120

Second, the test does visit `/services` right after the login, line 19. This test was introduced to simply validate that the page was added as per the requirements of the initial skeleton work. Its the minimal test required to ensure the page is there. This will be fleshed out the more functionality we add to the page.

Thirdly, you're right its likely not necessary to clean up the test session. I'll remove that in AB#15120.